resources/views/product/partials/quantity.blade.php

```diff
@@ -1,12 +1,10 @@
-<x-rapidez::select
+<x-rapidez::input.select
     name="qty"
     label="Quantity"
     v-model="addToCart.qty"
-    class="w-auto"
-    labelClass="flex items-center sr-only mr-3"
-    wrapperClass="flex"
```

Comment:
Maybe a search through the project to check if we don't have those anymore?

Reply:
Done nothing found anymore after removing this.